tools/integration_tests/rapid_appends/rapid_appends_test.go

```diff
@@ -129,6 +129,40 @@ func (t *RapidAppendsSuite) TestAppendsAndRead() {
 	}
 }
 
+func (t *RapidAppendsSuite) TestAppendSessionInvalidatedByAnotherClientUponTakeover() {
+	t.SetupSubTest()
+	defer t.TearDownSubTest()
+
+	// Initiate an append session using the primary file handle opened in append mode.
+	appendFileHandle := operations.OpenFileInMode(t.T(), path.Join(primaryMntTestDirPath, t.fileName), os.O_APPEND|os.O_WRONLY|syscall.O_DIRECT)
+	defer func() {
+		// Closing file handle purely for resource cleanup.
+		// FlushFile() triggered on Close() is expected to error out with stale NFS file handle.
```

Comment:
you can inline the defer call since you have only one line in defer fun()
i.e, defer operations.CloseFileShouldThrowError(t.T(), appendFileHandle)